### ISSUE TITLE
Still not able to debug in Chrome

### ISSUE THREAD

**Issue by peterpeterparker** [driftyco/ionic-app-scripts#143]
Hi,

I upgraded to the last version of ionic-app-scripts 0.0.30 but I'm still unable to debug in Chrome. When I try:

- in a provider, the breakpoints are reached, but the values/variables/etc. are wrongly estimated. The debugger show me variables as "undefined" where in fact they aren't (I set console.log to double check)

- in my pages, when I set breakpoints in ionViewWillEnter(), these are never reached aka it always went thru without stopping

My package.json is the following: 

     {
       "compilerOptions": {
         "allowSyntheticDefaultImports": true,
         "declaration": true,
         "emitDecoratorMetadata": true,
         "experimentalDecorators": true,
         "sourceMap": true,
         "lib": [
           "dom",
           "es2015"
         ],
         "module": "es2015",
         "moduleResolution": "node",
         "target": "es5",
         "removeComments": true
       },
       "exclude": [
         "node_modules"
       ],
       "compileOnSave": false,
       "atom": {
         "rewriteTsconfig": false
       }
     }

Could you have a look at that issue plz?

Thx in advance

**Comment by adamdbradley** [comment]
I can't replicate this behavior. @jgw96 are you seeing the same issues?

**Comment by jgw96** [comment]
Hello @peterpeterparker ! Thanks for opening an issue with us. Unfortunately I cannot seem to reproduce this behavior either with my test app. Do you have a repo that i can reproduce this issue with?

**Comment by peterpeterparker** [comment]
Thx guys for having a look at my issue.

@jgw96 my project is on a prive repo but I noticed that I've got exactly the same behavior problem with the Ionic2 starter app I checked out when RC.0 was introduced.

I added that project to Github there

https://github.com/peterpeterparker/ionic2RC-test

The problem is most probably coming from an error from my side and I thank you in advance for your understanding.

P.S.: To reproduce the behavior

- ionic serve
- chrome debug
- break point in "about.ts" in ionViewWillEnter

=> on my laptop, the debugger didn't stopped there
